try:
    print(a)
except NameError as erro:
    print("name:", "dia")